tutorials/pl/basic-snap-usage-pl/basic-snap-usage.md

```diff
@@ -0,0 +1,329 @@
+
+### Instalacja i uruchomienie
+
+Jak możesz zauważyć, wielu twórców udostępniło snapy typu „hello world”. Zainstalujmy jeden z nich.
+
+```bash
+$ sudo snap install hello
+
+hello (stable) 2.10 from 'canonical' installed
+```
+
+positive
+: Informacja: kiedy zainstalujesz pierwszego snapa, snapd pobierze postawowy podsystem snapa (który zawiera niewielką liczbę aplikacji o wadze do kilku megabajtów). W rezultacie, czas pobrania pierwszego snapa może być trochę dłuższy niż w przypadku kolejnych.
+
+
+Aby uruchomić pakiet, wprowadź hello w wiersz poleceń (jest to polecenie dostarczone przez snap `hello`:
+
+```bash
+$ hello
+Hello, world!
+```
+
+Oczywiście to tylko prosty przykład, który ma cechy wspólne dla wszystkich snapów, więc jest odizolowany i ma dostęp tylko do własnych danych.
+
+### Sprawdź, które snapy są zainstalowane
+
+Aby wyświetlić listę zainstalowanych snapów, wprowadź `snap list`, które poda również wersję programu, autora i dodatkowe informacje.
+
+
+```bash
+$ snap list
+Name             	Version	Rev  Developer   Notes
+hello            	2.10   	20   canonical   -
+core      	16.04.1	423  canonical   -
+```
+
+Zauważ, że pakiet `core` zawierający podstawowy podsystem również znajduje się na liście. Zawiera on również aktualna wersję snapd, dzięki czemu będzie on zawsze aktualny. 
+
+### Uaktualnianie systemu
+
+Będzie dla Ciebie dobrą wiadomością, że snapy są aktualizowane automatycznie codziennie. Jeżeli chcesz, możesz aktualizować snapy ręcznie poleceniem `snap refresh`. Zaktualizuje to wszystkie Twoje snapy, chyba że określisz konkretny pakiet w poleceniu.
+
+
+```bash
+$ sudo snap refresh hello
+error: cannot refresh "hello": snap "hello" has no updates available
+$ sudo snap refresh
+core updated
+hello 64.75 MB [=====================================>___]   12s
+
+```
+
+Nie są to wszystkie funkcje polecenia snap: możesz bardziej dokładnie śledzić wersje aplikacji, w zależności od tego, jak bardzo chcesz być na bieżąco z nową funkcjonalnością. Jest to możliwe dzięki funkcji kanałów. Możemy zagłębić się w to później.
+
+## Więcej możliwości snapów
+Duration: 1:00
+
+### Snapy mogą dodawać więcej niż jedno polecenie
+
+Nasz pierwszy przykład był prosty i zawierał tylko jedno polecenie, ale zwykle pakiety snap zawierają więcej niż jedno polecenie (na przykład zestaw narzędzi umieszczony w jednym snapie). Wszystkie te polecenia należą do przestrzeni nazw o nazwie pakietu. Wykonaj następujące kroki, aby zobaczyć przykład tego w działaniu:
+
+
+```bash
+$ sudo snap install hello-world
+0 B / 20.00 KB [_______________________________________________________] 0.00 %
+
+hello-world (stable) 6.3 from 'canonical' installed
+$ hello-world
+Hello World!
+$ hello-world.env
+< env variables >
+```
+
+
+positive
+: Informacja: pierwsze polecenie nie posiada przedrostka. To dlatego, że snapy mogą posiadać jedno domyślne polecenie.
```

Comment:
Let's highlight that this `domyślne polecenie` must be exactly the same as the name of the snap itself.